### ISSUE TITLE
[Suggestion] Move html markup to separate template files

### ISSUE THREAD

**Issue by coreymckrill** [wceu/camptix-invoices#28]
Throughout this plugin there are render callback functions that include big blocks of code that look something like this:

```
echo '<h3>' . esc_html__( 'Order details', 'invoices-camptix' ) . '</h3>';
$item_line = '<tr>
	<td>%1$s</td><!-- name -->
	<td>%2$.2f</td><!-- price -->
	<td>%3$s</td><!-- qty -->
	</tr>';
vprintf(
	'<table class="widefat"><thead><tr>
	<th>%1$s</th>
	<th>%2$s</th>
	<th>%3$s</th>
	</tr></thead><tbody>',
	array(
		esc_html__( 'Title', 'invoices-camptix' ),
		esc_html__( 'Unit price', 'invoices-camptix' ),
		esc_html__( 'Quantity', 'invoices-camptix' ),
	)
);
```

As someone who is unfamiliar with this code, this is really hard to follow. In the interest of long-term maintainability by multiple developers, readability is important. One possible way to improve readability would be to create a separate file for each of the metaboxes and other areas that need to be rendered, minimizing the amount of non-HTML logic that they contain, and load them within the render callback via `include` or `require`. Here are some examples from elsewhere in the WordCamp codebase:

* Meetup report
  * [Render callback](https://meta.trac.wordpress.org/browser/sites/trunk/wordcamp.org/public_html/wp-content/plugins/wordcamp-reports/classes/report/class-meetup-groups.php#L279)
  * [Template file](https://meta.trac.wordpress.org/browser/sites/trunk/wordcamp.org/public_html/wp-content/plugins/wordcamp-reports/views/html/meetup-groups.php)
* Invoice metabox
  * [Render callback](https://meta.trac.wordpress.org/browser/sites/trunk/wordcamp.org/public_html/wp-content/plugins/wordcamp-payments/includes/sponsor-invoice.php#L290)
  * [Template file](https://meta.trac.wordpress.org/browser/sites/trunk/wordcamp.org/public_html/wp-content/plugins/wordcamp-payments/views/sponsor-invoice/metabox-general.php)

**Comment by avillegasn** [comment]
This is now fixed.
